src/comments.js

```diff
@@ -520,13 +520,20 @@ function handleTemplateLiteralComments(enclosingNode, comment) {
       comment
     );
     // Enforce all comments to be leading block comments.
-    comment.type = "CommentBlock";
+    makeBlockComment(comment);
     addTrailingComment(enclosingNode.expressions[expressionIndex], comment);
     return true;
   }
   return false;
 }
 
+function makeBlockComment(comment) {
+  if (!util.isBlockComment(comment)) {
+    comment.value += " ";
+  }
+  comment.type = "CommentBlock";
```

Comment:
Yeah, I can probably take a look at that at some point. For now, though, what if I remove this part, so we can merge in the helper function extraction without changing behavior?

```js
   if (!util.isBlockComment(comment)) {
     comment.value += " ";
   }
```